_posts/pl/2021-09-31-zarzadzanie-stanem-aplikacji-frontendowej.md

```diff
@@ -218,7 +218,7 @@ Należy również zaznaczyć, że Redux nie jest jedynym sposobem na zarządzani
 ## Podsumowanie
 Łatwo zauważyć, że na pierwszy ogień NgRx, czy szerzej - Redux - potrafią nieco przytłoczyć ilością kodu, którą trzeba napisać, by nawet drobne funkcjonalności działały. Jest to jeden z największych zarzutów wobec tego rozwiązania, więc jeśli takie były Twoje odczucia podczas czytania tego artykułu - gratuluję krytycznego myślenia! Zauważmy jednak, że po przebrnięciu przez początkowe trudności zostajemy z aplikacją, którą bardzo łatwo możemy otestować, rozszerzać i której działanie jest jasno zdefiniowane.
```

Comment:
Luźna sugestia 2 zdanie: 
Jest to jedna z największych słabości/ Jest to jeden z największych mankamentów tego rozwiązania...

Reply:
Tutaj pozwolę się nie zgodzić z tego względu, że to nie chcę tutaj wartościować, a jedynie wskazać, że takie są opinie

Reply:
Jasna sprawa. Zgadzam się jak najbardziej :)